index.html

```diff
@@ -0,0 +1,140 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+ </head>
+ <body>
+  <header>   
+    <div class="mylogo">
+      <a href="#">
+        My Logo
+      </a>
+    </div>
+    <nav class="hum-menu">
+        <svg
+          width="40"
+          height="40"
+          viewBox="0 0 40 40"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"
+            fill="#6070FF"/>
+        </svg>
+    </nav>
+ </header>
+
+ <section class="headerline">
+    <div class="headerline-detail">
+        <div>
+            <h1 class="name">I'm Marine Glad to see you!</h1>
+        </div>
+        <div>
+            <p class="aboutme">
+                I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+            </p>
+        </div>
+        <div>
+            <p class="connect">Let's Connect</p>
+        </div>
+        <div>
+            <ul class="social-media-icons">
+                <li>
+                    <a href="https://twitter.com">
+                        <svg 
+                          width="20"
+                          height="20"
+                          viewBox="0 0 20 20"
+                          fill="none"
+                          xmlns="http://www.w3.org/2000/svg"
+                        >
+                        <path
+                          d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10204 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253176 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
+                          fill="#505F79"
+                          />
+                        </svg>
```

Comment:
- [ ] to match the Figma design kindly add the margin-bottom below the social icons and add the correct border-radius for the bottom left corner 🙏 
![image](https://user-images.githubusercontent.com/89282221/198285742-d3e75e0f-7c67-4724-a7c3-b84fa73edaaa.png)